content/applications/finance/accounting/receivables/customer_invoices/cash_discounts.rst

```diff
@@ -1,68 +1,180 @@
-====================
-Offer cash discounts
-====================
-
-Cash discounts are incentives you can offer to customers to motivate
-them to pay within a specific time frame. For instance, you offer a 2%
-discount if the customer pays you within the first 5 days of the
-invoice, when it is due in 30 days. This approach can greatly improve
-your average collection period.
-
-Set up a cash discount
-======================
-
-To set up a cash discount, go to :menuselection:`Accounting -->
-Configuration --> Management --> Payment Terms` and click on
-*Create*. Add a *Percent* type of term with a corresponding value
-(e.g. 98% of the total price for a 2% discount) and the number of days
-during which the offer is valid. You can also change the default balance
-term if needed.
-
-.. image:: cash_discounts/cash_discounts01.png
-  :align: center
-
-Start offering the cash discount
 ================================
+Cash discounts and tax reduction
+================================
+
+**Cash discounts** are reductions in the amount a customer must pay for goods or services offered as
+an incentive for paying their invoice promptly. These discounts are typically a percentage of the
+total invoice amount and are applied if the customer pays within a specified time. The purpose of
+offering cash discounts is to encourage customers to pay their bills quickly, which can help the
+company maintain a steady cash flow.
+
+.. example::
+   You issue a €100 invoice on the 1st of January. The full payment is due within 30 days, and you
+   also offer a 2% discount if your customer pays you within seven days.
+
+   The customer can pay €98 up to the 8th of January. After that date, they would have to pay €100
+   by the 31st of January.
+
+A :ref:`tax reduction <cash-discounts/tax-reductions>` can also be applied depending on the country
+or region.
+
+.. _cash-discounts/configuration:
+
+Configuration
+=============
+
+To grant cash discounts to customers, you must first set up the :ref:`type of tax reduction
+<cash-discounts/tax-reductions>`, verify the :ref:`gain and loss accounts
+<cash-discounts/gain-loss-accounts>`, and configure new :ref:`payment terms
+<cash-discounts/payment-terms>`.
+
+.. _cash-discounts/tax-reductions:
+
+Tax reductions
+--------------
+
+Depending on the country or region, the base amount used to compute the tax can vary, which can lead
+to a **tax reduction**.
+
+To configure how the tax reduction is applied, go to :menuselection:`Accounting --> Configuration
+--> Settings`, and in the :guilabel:`Cash Discount Tax Reduction` feature, select one of the three
```

Comment:
```suggestion
--> Settings`, and in the :guilabel:`Cash Discount Tax Reduction` feature, select one of the three
```
Where is that feature ? (I know where it is, and I know I can search for it, but not everybody does)